### PR TITLE
Update aks SURE-6882.md

### PR DESCRIPTION
**Pull request by davidstauffer**
Fixes #[[SURE-6882](https://jira.suse.com/browse/SURE-6882)]

## Reminders

- See the [README](../README.md) for more details on how to work with the Rancher docs.

- Verify if changes pertain to other versions of Rancher. If they do, finalize the edits on one version of the page, then apply the edits to the other versions.

- If the pull request is dependent on an upcoming release, make sure to target the release branch instead of `main`.

### CODE REVIEW

**Review by martyav — COMMENTED** [review]
On the ticket, it's suggested that we model the update after the existing page, https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks#minimum-eks-permissions. 

I think that should include positioning the equivalent sections in the same places as on https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks. This makes it easier to browse the docs, as similar pages have similar sections in the same locations. Sections shouldn't be organized in the chronological order that we add them.

So, when updating this page, "Programmatically Creating AKS Clusters" should remain the last section on the page, and "Setting Up the Minimum Permission Role with the Azure Command Line Tool" should come before "Syncing", to match the order on the EKS page. 

Also, the sections "Setting Up the Resource Group with the Azure Command Line Tool" and "Setting Up the Service Principal with the Azure Command Line Tool" should be moved up. There is already a section entitled "Setting Up the Resource Group with the Azure Command Line Tool" (https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks#setting-up-the-service-principal-with-the-azure-command-line-tool), so the new section being added needs to specify that it's for custom Azure roles. "Setting Up the Service Principal with the Azure Command Line Tool" should come after  "Setting Up the Resource Group with the Azure Command Line Tool" and include some mention that it's for Azure custom roles.

File: docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md
Comment (mjura) [inline docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md:220]: Please add also
```suggestion
        "Microsoft.ContainerService/managedClusters/delete",
```

**Review by mjura — CHANGES_REQUESTED** [review]
please add one more permission

File: docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md
Comment (mjura) [inline docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md:219]: It seems that `write` permission is gone, please revert it

```suggestion
        "Microsoft.ContainerService/managedClusters/read",
        "Microsoft.ContainerService/managedClusters/write"
```

**Review by mjura — CHANGES_REQUESTED** [review]
It seems that write permission is gone, please revert it
```
"Microsoft.ContainerService/managedClusters/write"
```

**Comment by martyav** [comment]
@davidstauffer I just pushed a commit to re-organize the page. I also removed a duplicate section and added the suggestion proposed by @mjura re: "Microsoft.ContainerService/managedClusters/write" permissions

**Comment by davidstauffer** [comment]
Thanks @martyav , are we ready to merge? :)

File: docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md
Comment (martyav) [inline docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md:257]:
```suggestion
2. Apply the Rancher AKSv2 Role:
```

File: docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md
Comment (martyav) [inline docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md:263]:
```suggestion
3. Verify if the Rancher AKSv2 Role was created:
```

**Review by martyav — CHANGES_REQUESTED** [review]

**Comment by martyav** [comment]
@davidstauffer one last thing -- does this apply to any other versions besides 2.7?

**Comment by davidstauffer** [comment]
Yes, in theory, it should apply as well to previous versions.  This is setting the privileges on the Azure side.  Better to double check with @mjura , if this can be applied to past versions...

**Comment by davidstauffer** [comment]
I would suggest we merge for now at least to get it in for 2.7. I think our users would highly benefit from the improved docs around that.

**Comment by martyav** [comment]
@davidstauffer Let me just add the versioned 2.7 page

**Comment by martyav** [comment]
@davidstauffer I'm getting an error when I try to push to your branch. Could you copy the file over to versioned_docs/version-2.7/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md

**Comment by davidstauffer** [comment]
I added it as well to the versioned docs. Let me know if that works for you. David

**Comment by martyav** [comment]
@davidstauffer I think we're ready now. We just need approval from @mjura. You've already addressed the changes requested from their review. If this is a time sensitive ticket I'll dismiss their review as stale, so we can merge.

File: docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md
Comment (mjura) [inline docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md:250]: Here `f5e1bf9e-ec79-4fc5-8354-53e2fcc0d99f` it should be removed, here should be SUBSCRIPTION_ID

File: versioned_docs/version-2.7/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md
Comment (mjura) [inline versioned_docs/version-2.7/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md:250]: Here `f5e1bf9e-ec79-4fc5-8354-53e2fcc0d99f` it should be removed, here should be SUBSCRIPTION_ID

**Review by mjura — DISMISSED** [review]
please remove SUBSCRIPTION_ID from role

**Comment by davidstauffer** [comment]
Done @mjura . I think we can now merge @martyav .
Many thanks,
David

**Pull request merged**
